tests/e2e/kueue_test.go

```diff
@@ -164,6 +172,36 @@ func (tc *KueueTestCtx) testOwnerReferences() error {
 	return nil
 }
 
+func (tc *KueueTestCtx) validateVAPReady() error {
+	// if ocp is 4.17+ then VAP and VAPB should be created
+	if cluster.GetClusterInfo().Version.GTE(semver.MustParse("4.17.0")) {
```

Comment:
this check shold probably me moved to the `t.Run` block, like:

```go
t.Run("Validate Kueue Dynamically create VAP", func(t *testing.T) {
        if !cluster.GetClusterInfo().Version.GTE(semver.MustParse("4.17.0") {
            t.Skip("Disabled as requires OpenShift >= 4.17")
            return
        }
        
	err = kueueCtx.validateVAPReady()
	require.NoError(t, err, "Kueue instance is not Ready")
})
```

Reply:
yep, doing rebase now, need local test before push the final change